user/details/style.css

```diff
@@ -278,3 +280,124 @@ footer p a {
     height: 20px;
   }
 }
+
+.user-pr {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 1rem 0 1rem;
+}
+
+.user-pr > h2 {
```

Comment:
Use `class` for styling pr-title or heading.

Reply:
done